bd-logger/src/log_replay.rs

```diff
@@ -251,6 +254,12 @@ impl ProcessingPipeline {
         );
       }
 
+      if result.take_screenshot {
+        if let Err(e) = self.take_screenshot_tx.try_send(()) {
```

Comment:
Now that you have the concurrency control it's probably fine to use normal send here and not try send? In practice it probably doesn't matter but if you keep try_send I would add a comment on why.